src/Service/SharedManagementService.cs

```diff
@@ -142,8 +144,8 @@ public async Task<CreateAppResultDto> GenerateAccount(string appId, CreateAppDto
     public async Task<ValidateSecretKeyDto> ValidateSecretKey(string secretKey)
     {
         var appId = GetAppId(secretKey);
-        var storage = tenantFactory.Create(appId);
 
+        var storage = _globalStorageFactory.Create();
```

Comment:
non-blocking, separate-issue: This might be the subject of a separate PR and issue but: I can't see why we need a globalStorageFactory instead of injecting it?

Reply:
I think I did it for consistency, but I can look into it.